ConsoleApp/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Text.RegularExpressions; 			
+public class Program
+{
+	public static void Main()
+	{
+		SumNumbers();
+	}
+	
+	private static void SumNumbers()
+	{
+		int fNumber, sNumber;
+		Console.WriteLine("\nPlease, enter the first number: ");
+		do
+		{
+			var input1 = Console.ReadLine();
+			if (Regex.IsMatch(input1, @"^\d+$")) // ^ beginning of string, \d single digit, $ end of string
```

Comment:
no need to use regex, you can simply use `int.TryParse`. And your code won't work with negative numbers